catalog/dags/common/storage/media.py

```diff
@@ -304,17 +308,22 @@ def _format_raw_tag(self, tag: str) -> dict:
 
     def _validate_filetype(self, filetype: str | None, url: str) -> str | None:
         """
-        Extract filetype from the media URL if filetype is None.
+        Extract filetype from the media URL if filetype is None and check
+        that it corresponds to the media type of the class, while also unifies
```

Comment:
It's better to split this into two sentences, because there's a mismatch now (first part of the sentence is imperative, "extract", and the second is not, "unifies")

Reply:
Fixed the grammar! Good eye 😆 Thanks!